CONTRIBUTING.md

```diff
@@ -1,108 +1,130 @@
 
-We use GitHub issues to track bugs exclusively. We have documented our issue handling processes in the [Issues wiki](https://github.com/facebook/react-native/wiki/Issues).
+**Git Command Line**
+If you are using the command line, you will want to make sure you have your branch locally. It takes time for it to show up automatically, so you can make it show by running `git remote update origin –prune` in your command prompt from the root. Run `git branch -a` to see if your new branch shows up. Now you will want to check out your branch locally. You can do this with `git checkout -b branch-name`. Confirm you are now working out of the branch with `git branch`.
 
-### Security Bugs
+### Merging upstream master into your fork master
 
-Facebook has a [bounty program](https://www.facebook.com/whitehat/) for the safe disclosure of security bugs. In those cases, please go through the process outlined on that page and do not file a public issue.
+From time to time, your fork will get out of sync with the upstream remote. Use the following commands to get the master branch of your fork up up to date.
 
-## Helping with Documentation
+```
+> git fetch upstream
+> git checkout master
+> git pull upstream master
+> git push
+```
 
-The React Native documentation is hosted as part of the React Native website repository at https://github.com/facebook/react-native-website. The website itself is located at <https://facebook.github.io/react-native> and it is built using [Docusaurus](https://docusaurus.io/). If there's anything you'd like to change in the docs, you can get started by clicking on the "Edit" button located on the upper right of most pages in the website.
+### Merging upstream master into your current branch
 
-If you are adding new functionality or introducing a change in behavior, we will ask you to update the documentation to reflect your changes.
+Use these commands instead if you would like to update your *current* branch in your fork from the upstream remote.
 
-### Contributing to the Blog
+```
+> git fetch upstream
+> git pull upstream master
+> git push
+```
+## Contributing
 
-The React Native blog is generated [from the Markdown sources for the blog](https://github.com/facebook/react-native-website/tree/master/website/blog).
+### Make the fix
+Now that your branch is set up and ready for commits, go ahead and fix the bug you are working on or make some small change that you want to check in. 
+ 
+### Verify your changes
+Manually test your fix, and ensure that all tests are in a good state
 
```

Comment:
Should we describe running the local automated test that will be run in CI when they make a pull request?
i.e.
yarn test  # run jest tests on JavaScript
yarn lint # run eslint on JavaScript
yarn flow-check-macos # run Flow checks on JavaScript

Manually test by running RNTester
Run Unit Tests and Integration Tests in the RNTesterPods Xcode project.

Reply:
Maybe TMI as if they will be caught by CI during the PR

Reply:
It is good to add these as best practices with a call out that it will be caught in CI. I will add.